core/controllers/ingest.py

```diff
@@ -313,3 +299,59 @@ def save_submission_file(excel_file, submission_id):
     submission.submission_file = excel_file.stream.read()
     db.session.add(submission)
     db.session.commit()
+
+
+def get_programmes_same_round_or_older(reporting_round: int, programme_ids: list[str]) -> dict[str, str]:
+    """Returns list of programme ids of the same round or older than the reporting round.
```

Comment:
Can this docstring be a bit more specific? The programme id's themselves are not of a reporting round, but via a join to project and submission. 

Something like:
Return ids of programmes updated in any round up to and including the specified round